index.html

```diff
@@ -6,301 +6,195 @@
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
-                    </form>
-                </div>
-                <div class="touch-class3">
-                    <form>
-                        <table class="form-table">
-                            <tr class="form-tr couple-tr">
-                                <td class="form-td"><input type="text" placeholder="Full Name"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td align="center"><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
-                    </form>
-                </div>
+            <div class="desktop-form">
+                <form action="https://formspree.io/f/moqzdnpo" method="post">
+                    <input type="text" name="first-name" placeholder="First Name">
+                    <input type="text" name="last-name" placeholder="Last Name">
+                    <input id="email" name="e-mail" type="email" placeholder="E mail address">
+                    <td colspan="2" name="message" class="form-td"><textarea>Typing something</textarea>
+                    <button class="see-project-btn">Get in touch</button>
+                </form>
             </div>
-        </div>
-    </section>
+            <div>
+                <form class="mobile-form" action="https://formspree.io/f/moqzdnpo" method="post">
+                    <input type="text" name="full-name" maxlength="30" required 
+                         placeholder="Full Name">
+                    <input id="email" name="e-mail" required type="email" placeholder="Email address">
+                    <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+                    <button class="message-submit-btn">Get in touch</button>
+                </form>
+            </div>
```

Comment:
- [ ] [ OPTIONAL ] Good job implementing the requested changes. But as the previous reviewer had requested about the input focus, you have implemented the wrong colors. Please note that all your form inputs' borders should be `#d0d9d4`, and on focus they should change to `#60c095;`